lib/internal/process.js

```diff
@@ -1,6 +1,8 @@
 'use strict';
 
 const errors = require('internal/errors');
+const util = require('util');
```

Comment:
It seems that the intention in this file was to do this require only when really needed so I'm not sure if it's a good idea to require it unconditionally.